build-bin/configure_docker_push

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Copyright 2019-2020 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+set -ue
+
+# This script sets up anything needed for `./docker_push`.
+#
+# This script primarily helps `.travis.yml` and `RELEASE.md` contain nearly the same contents, even
+# if certain OpenZipkin projects need slight adjustments here.
+
+# openzipkin-contrib is named differently in docker.io (openzipkincontrib). Skip the latter for now.
+export DOCKER_RELEASE_REPOS=ghcr.io
```

Comment:
this was a silly mistake, corrected in e55c46ba17fc3a910015739227acb3d6da51ac68

exporting variables in a subshell intended for another script is frivolous :P